internals/webpack/webpack.config.prod.js

```diff
@@ -87,7 +87,7 @@ module.exports = require('./webpack.config.base')({
       AppCache: {
         events: true
       },
-      relativePaths: isProd() || isUAT(),
+      relativePaths: false,
```

Comment:
Why is this removed, shouldn't we be setting relative paths?

Reply:
But completely getting rid of them would break if this was to be built on PROD or UAT, or would it not? Can you try building and see if everything is working or not?